tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyMain.java

```diff
@@ -3701,6 +3701,10 @@ public JSONObject statusJSON(HttpServletRequest request) throws JSONException {
         private int users = CookieSession.getUserCount();
         private String sessionMessage = null;
 
+        private Runtime r = Runtime.getRuntime();
+        double memtotal = r.totalMemory() / 1024000.0;
+        double memfree = r.freeMemory() / 1024000.0;
+
         private JSONObject toJSONObject() throws JSONException {
```

Comment:
how about adding a comment about prometheus here? Otherwise it's natural to assume Survey Tool is the only client for status requests, and someone will just delete these again in the belief that they're dead code